torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -254,7 +254,10 @@ kir::ExpressionEvaluator bindKernelInputs(
           expr_eval.bind(extent, value);
         }
       }
-    } else if (input->isScalar() && input->dtype() == DataType::Int) {
+    } else if (input->isScalar() && input->dtype() == DataType::Int) { // NOLINT
```

Comment:
did clang-format split the comment over 4 separate lines?

Reply:
Yes, it seems so.

Reply:
Changed to use `NOLINTNEXTLINE`.